indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -273,8 +273,9 @@ class RHRegistrationCreate(RHManageRegFormBase):
     def _get_user_data(self, user):
         if user is None:
             return {}
-        user_data = {t.name: getattr(user, t.name, None) for t in PersonalDataType}
-        user_data['title'] = get_title_uuid(self.regform, user_data['title'])
+        user_data = {t.name: getattr(user, t.name, t.default)
+                     if user else t.default for t in PersonalDataType}
```

Comment:
I really don't like the wrapping  here :/

```python
user_data = {t.name: getattr(user, t.name, t.default) if user else t.default
                     for t in PersonalDataType}
```

or since it fits in one line:

```python
user_data = {t.name: getattr(user, t.name, t.default) if user else t.default for t in PersonalDataType}
```